README.md

```diff
@@ -1 +1,62 @@
-# Hello
\ No newline at end of file
+<h3 align="center">Medium Scraper-Ruby</h3>
+
+<div align="center">
+
+[![Status](https://img.shields.io/badge/status-active-success.svg)](https://github.com/umairarshadbutt/Hello/)
+[![Github Issues](https://img.shields.io/badge/GitHub-Issues-orange)](https://github.com/umairarshadbutt/Hello//issues)
+[![GitHub2 Pull Requests](https://img.shields.io/badge/GitHub-Pull%20Requests-blue)](https://github.com/umairarshadbutt/Hello//pulls)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](/LICENSE)
+[![Linkedin](https://img.shields.io/badge/-LinkedIn-blue?style=flat&logo=Linkedin&logoColor=white)](https://www.linkedin.com/in/umair-arshad-butt/)
+[![Gmail](https://img.shields.io/badge/-Gmail-c14438?style=flat&logo=Gmail&logoColor=white)](mailto:Umair,umair6622@gmail.com)
+[![Portfolio](https://img.shields.io/badge/umair--arshad-Portfolio-brightgreen)](https://umairarshad.me/)
+
+</div>
+<p align="center">The main goal is to implement the Simple HTML file template.</p>
+![screenshot](assets/images/screen-shot.png)
```

Comment:
[][REQUIRED] The screenshot for this project isn't present. Please make sure to add it for a more professional-looking readme.